src/components/Homepage/Carousel/Item.tsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import tickIcon from "../../../assets/icons/tick.svg?react";
+import "./ItemFrame.css";
+
+interface BaseItem {
+  type: string;
+  source: string;
+  bottomWidthRem?: number;
+  paddingRem?: number;
+}
+
+interface TextItem extends BaseItem {
+  type: "text";
+  message: string;
+}
+
+interface ImageItem extends BaseItem {
+  type: "image";
+  imageSrc: any;
+  imageAlt: string;
+}
+
+type CarouselItemProps = TextItem | ImageItem;
+
+const CarouselItem: React.FC<CarouselItemProps> = ({
+  type,
+  source,
+  bottomWidthRem = 1.75,
+  paddingRem = 2.5,
+  ...rest
+}) => {
+  const message = type === "text" ? (rest as TextItem).message : "";
+  const imageSrc = type === "image" ? (rest as ImageItem).imageSrc : "";
+  const imageAlt = type === "image" ? (rest as ImageItem).imageAlt : "";
+
+  const renderImage = () => {
+    if (!imageSrc) return null;
+
+    if (typeof imageSrc === "function") {
+      return React.createElement(imageSrc, { "aria-label": imageAlt });
+    }
+
+    return (
+      <img
+        src={imageSrc as any}
+        alt={imageAlt}
+        className="max-w-full h-auto"
+        onError={(e) => console.error("Image failed to load:", e)}
+      />
+    );
+  };
```

Comment:
This is too complicated and contains one branch of code that I'm sure is never executed, compounded by the switch from `ImageMetadata` to `any`.

This should simply be something like:

```suggestion
  const renderImage = (image: ImageItem) => {
    return (
      <img
        src={image.imageSrc}
        alt={image.imageAlt}
        className="max-w-full h-auto"
      />
    );
  };
```

So you might not even need a function for this.

Reply:
We still need the function because when you import SVGs in React components they are functions.

Reply:
If that's so, when is the following used then? As I understand it, all of our images are SVGs, so `typeof imageSrc === "function")` would be true 100% of the time.

```
    return (
      <img
        src={imageSrc as any}
        alt={imageAlt}
        className="max-w-full h-auto"
        onError={(e) => console.error("Image failed to load:", e)}
      />
    );
```

Reply:
@gnarea That code is for the GIF, that is considered as image

Reply:
Oh, never mind, I forgot that we also have a gif.